full-backend-tests/src/test/java/org/graylog/datanode/RemoteReindexingMigrationIT.java

```diff
@@ -84,15 +84,16 @@ void testRemoteAsyncReindexing() throws ExecutionException, RetryException {
 
         final String request = """
                 {
+                    "allowlist": "%s",
                     "hostname": "%s",
                     "indices": ["%s", "%s"],
                     "synchronous": false
                 }
-                """.formatted(openSearchInstance.internalUri(), indexName, indexName2);
+                """.formatted(allowlistValue(openSearchInstance.internalUri()), openSearchInstance.internalUri(), indexName, indexName2);
```

Comment:
maybe we should also add a check in `RemoteReindexingMigrationAdapterOS2#prepareCluster` to avoid the npe

Reply:
Added a check